lib/check-port.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
```

Comment:
This is not needed because this function is not going to be executed directly like `build-files.js`